{{cookiecutter.project_slug}}/requirements.txt

```diff
@@ -1 +0,0 @@
--r requirements/prod.txt
```

Comment:
I'd prefer to leave this file here, it is a good shortcut for people used to just run `pip install -r requirements`